app/models/event.rb

```diff
@@ -1,24 +1,14 @@
 class Event < ApplicationRecord
   belongs_to :place 
   belongs_to :user
-<<<<<<< HEAD
-  has_and_belongs_to_many :members, class_name: "User", dependent: :destroy 
-=======
-  has_and_belongs_to_many :members, class_name: "User"
+  has_and_belongs_to_many :members, class_name: "User", before_remove: notice_before_remove
```

Comment:
From what I remember `before_remove` model relation doesn't have this kind of method.